tox.ini

```diff
@@ -128,7 +127,7 @@ deps =
 # distribution build.
     !ddtracerun: wrapt
     !msgpack03-!msgpack04-!msgpack05-!ddtracerun: msgpack-python
-    pytest>=3
+    pytest>=3.0.0,<4.0.0
```

Comment:
That looks wrong.

Reply:
yup, you are correct, I messed up the merge conflict I had, great catch!

Reply:
```suggestion
    pytest>=3.0.0
```